demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -897,6 +897,32 @@ def test_validate_no_old_format__without_toversion(self, mocker):
         assert not validate_manager.validate_no_old_format(old_format_files)
         assert handle_error_mock.call_count == 2
 
+    def test_validate_no_old_format_deprecated_content(self, repo):
```

Comment:
please also add test for script for script/integration without the `deprecated` field in the yml